lib/charliecloud.py

```diff
@@ -411,10 +411,12 @@ def DEBUG(msg, hint=None, **kwargs):
    if (verbose >= 2):
       log(msg, hint, None, "38;5;6m", "", **kwargs)  # dark cyan (same as 36m)
 
-def ERROR(msg, hint=None, trace=None, **kwargs):
-   log(msg, hint, trace, "1;31m", "error: ", **kwargs)  # bold red
+def ERROR(msg, hint=None, trace=None, fatal=False, **kwargs):
+   if (log_quiet < 3 or fatal):
```

Comment:
The `fatal` parameter seems awkward. What if we treat fatal errors and non-fatal errors (of which there are few) the same? Alternately, when we catch `Fatal_Error` we could just call `log()` manually.